domains/selva.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+      "username": "selvasankar",
+      "email": "selvasankar.p@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://selvasankar.github.io/"
```

Cannot access website, 404

CNAME can't contain protocol or `/`
```suggestion
      "CNAME": "selvasankar.github.io"
```